libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -174,9 +167,16 @@ proc init*(_: type[GossipSubParams]): GossipSubParams =
       pruneBackoff: 1.minutes,
       floodPublish: true,
       gossipFactor: 0.25,
+      d: 6,
```

Comment:
are this values hardcoded now?

Reply:
let's keep the const params and use those in here